AlphaWallet/Market/UniversalLinkHandler.swift

```diff
@@ -96,11 +96,17 @@ public class UniversalLinkHandler {
     
     //shortens price and expiry
     func formatMessageForLink(signedOrder: SignedOrder) -> String {
-        let message = signedOrder.message
+        //append encodable byte to differiantiate between spawnable and index
```

Comment:
This PR only makes generation of universal links include an additional byte, and not parse universal links with that additional byte?

The test UniversalLinkHandlerTests.testUniversalLinkParser() is failing.